client-app/src/desktop/AppModel.ts

```diff
@@ -21,7 +21,19 @@ import {panelsTab} from './tabs/panels/PanelsTab';
 import {fmtDateTimeSec} from '@xh/hoist/format';
 import {span} from '@xh/hoist/cmp/layout';
 import {BaseAppModel} from '../BaseAppModel';
+import {isEmpty} from 'lodash';
 
+// Encoding of json route params as base64
+export const routeParamEncoders = {
```

Comment:
This is kind of a funny thing to now have at the very top of our primary desktop AppModel, in terms of the being the first thing a reader sees.  Could they be moved to a utils file, or even just down near the bottom of this file?

Reply:
Couldn't find a relevant existing Utils file to add to, so just moved to the bottom.